1-exercises/H-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ let everyone = [
   "Swathi",
 ];
 
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive=everyone.slice(0,5); // complete this statement
+let lastFive=everyone.slice(-5); // complete this statement
 
 /* 
```

Comment:
I also like how you have commented most of the work, made easier for me follow what you have done.